src/Provider/Geonames/Geonames.php

```diff
@@ -92,6 +97,43 @@ public function reverseQuery(ReverseQuery $query): Collection
     }
 
     /**
+     * @param string|null $country
+     *
+     * @param string|null $locale
+     *
+     * @param int|null $maxRows
+     *
+     * @param int|null $startRow
+     *
+     * @return Collection
+     *
+     * @throws \Geocoder\Exception\Exception
+     */
+    public function getCountryInfo(string $country = null, string $locale = null, int $maxRows = null, int $startRow = null): Collection
```

Comment:
It is not passed, Geonames returns all countries [subjected to the limit (Default: 200)]. Let me know if we should ask for `$country`.